.github/workflows/publish.yml

```diff
@@ -16,14 +16,14 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 14
-          cache: 'yarn'
+          cache: 'npm'
           registry-url: 'https://registry.npmjs.org'
 
-      - name: Yarn Install
-        run: yarn install --network-concurrency 1
+      - name: NPM Install
+        run: npm ci --network-concurrency 1
```

Comment:
I'm fairly confident network concurrency is specific to yarn, so this might not work

Reply:
dropped it